code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -228,3 +231,51 @@
 	New()
 		..()
 		reagents.add_reagent("hyperzine", 10)
+
+/obj/item/weapon/storage/pill_bottle/time_release
+	name = "controlled release pill bottle"
+	desc = "A bottle containing special pills which can be calibrated for delayed release with sugar."
+
+/obj/item/weapon/storage/pill_bottle/time_release/New()
+	..()
+	for(var/i=0,i<7,i++)

```

Comment:
for future reference you can do ```for(var/i=0 to 7)```, quicker to write and tiny bit better performance. not a big deal though